verification/curator-service/src/controllers/home.ts

```diff
@@ -1,10 +1,37 @@
 import { Request, Response } from 'express';
+import axios from 'axios';
+
+interface Case {
+    _id: string;
+    outcome: string;
+    date: Date;
+}
+
+function getLatestCaseDate(cases: [Case]): Date {
```

Comment:
I think you can use Cases.find() and provide a {'sort': {'date': -1} param to do what you want

Reply:
As with all problems in JS, `reduce` is the simplest I can make this.